packages/blade/src/components/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,39 @@
+import type { ReactElement } from 'react';
+import type { IconComponent } from '..';
+
+import StyledIconButton from './StyledIconButton.web';
+import { makeAccessible } from '~src/utils/makeAccessible';
+
+export type IconButtonProps = {
+  /** Icon component to be rendered, eg. `CloseIcon` */
+  icon: IconComponent;
+  onClick: () => void;
+  size?: 'medium' | 'large'; // todo: discrepancy in size between code and figma
+  contrast?: 'low' | 'high';
+  /** **a11y**: Sets aria-label to help users know what the action does, eg 'Dismiss alert' */
+  accessibilityLabel: string;
```

Comment:
Let's format this correctly, and use a consistent jsdoc comment habit.

Reply:
Right, we can definitely consider that.

Though for this PR, to keep things scoped, I'll format the JSDocs correctly.